python3/totoro_agent/utils/util_functions.py

```diff
@@ -363,6 +362,50 @@ def is_movement(action):
     ]
     return action in actions
 
+# Update value map based on reward entities input
+def update_value_map(rval, value_map, world_dim, OUTER_MAP_VALUES=((-100, -100),(-100, -100))):
+    '''
+    Updates the reward value map with mask matrix application, based on reward entity.
+    Returns a numpy array representing the updated reward value map.
+    '''
+    # Add map padding
+    pad_dim = (world_dim[0]-1,world_dim[1]-1)
+    value_map = np.pad(value_map, (pad_dim,pad_dim), 'constant', constant_values=OUTER_MAP_VALUES)
+    rval_offset = pad_dim[0]-1
+
+    reward = rval[2]
+    reward_discount = reward/abs(reward)
+    rval_offset = 8 # padding offset
```

Comment:
should be world_dim[0] - 1 instead of hardcoded incase they update map dimension. Either that or hardcode the padding. Oh wait. You defined this above already and then overwritten it